scripts/optimize_png.js

```diff
@@ -12,6 +13,8 @@ const paths = [
   'src/static/images/buttons/',
   'src/static/images/email/',
   'src/static/images/icons/',
+  // 'src/public/images/mime-pdf' // crash with transparent
```

Comment:
Weird, it works fine for me with my own zopflipng build https://github.com/XhmikosR/zopfli/releases, which I have submitted to imagemin/zopflipng-bin here https://github.com/imagemin/zopflipng-bin/pull/15